src/plugins/tagfilter/tagfilter-en.hbs

```diff
@@ -0,0 +1,1278 @@
+					&lt;/li&gt;
+				&lt;/ul&gt;
+			&lt;/div&gt;
+		&lt;/div&gt;
+	&lt;/section&gt;</code></pre>
+</details>
+
+<section id="filteredAppsList" class="wb-tagfilter provisional">
+	<h2>Example 2 - Applications list</h2>
+	<div class="row">
+		<div class="col-sm-4 col-md-5 col-lg-6">
+			<div class="form-group">
+				<fieldset>
+					<legend class="h5 mrgn-tp-0"><span class="field-name">Institution</span></legend>
+					<select id="institution" class="full-width form-control wb-tagfilter-ctrl">
+						<option value="">All</option>
+						<option value="inst__EDC">Science and Economic Development Canada</option>
+						<option value="inst__AAFC">Agriculture and Agri-Food Canada</option>
+						<option value="inst__CBSA">Canada Border Services Agency</option>
+						<option value="inst__CBSAandPHAC">Canada Border Services Agency and Public Health Agency of Canada</option>
+						<option value="inst__CRA">Canada Revenue Agency</option>
+						<option value="inst__CSPS">Canada School of Public Service</option>
+						<option value="inst__ESDC">Employment and Social Development Canada</option>
+						<option value="inst__ECCC">Environment and Climate Change Canada</option>
+						<option value="inst__FOC">Fisheries and Oceans Canada</option>
+						<option value="inst__GAC">Global Affairs Canada</option>
+						<option value="inst__HC">Health Canada</option>
+						<option value="inst__IRCC">Immigration, Refugees and Citizenship Canada</option>
+						<option value="inst__ISC">Indigenous Services Canada</option>
+						<option value="inst__Innovation">Innovation</option>
+						<option value="inst__NDandCAF">National Defence and the Canadian Armed Forces</option>
+						<option value="inst__NFBC">National Film Board of Canada</option>
+						<option value="inst__NRC">Natural Resources Canada</option>
+						<option value="inst__PC">Parks Canada</option>
+						<option value="inst__PHAC">Public Health Agency of Canada</option>
+						<option value="inst__PSPC">Public Services and Procurement Canada</option>
+						<option value="inst__RCMP">Royal Canadian Mounted Police</option>
+						<option value="inst__SC">Statistics Canada</option>
+						<option value="inst__VAC">Veterans Affairs Canada</option>
+					</select>
+				</fieldset>
+			</div>
+		</div>
+		<div class="col-sm-8 col-md-7 col-lg-6">
+			<div class="form-group mrgn-bttm-0">
+				<fieldset>
+					<legend class="h5 mrgn-tp-0"><span class="field-name">Platform</span></legend>
+					<ul class="list-unstyled list-inline">
+						<li class="checkbox">
+							<label for="plat__ios">
+								<input type="checkbox" name="platform" id="plat__ios" class="wb-tagfilter-ctrl" value="plat__ios"> iOS
+							</label>
+						</li>
+						<li class="checkbox">
+							<label for="plat__android">
+								<input type="checkbox" name="platform" id="plat__android" class="wb-tagfilter-ctrl" value="plat__android"> Android
```

Comment:
What is the risk of having this filter control to be mixed with an external form control? Like have 2 control sibling with different purpose?

If all controls is about tag filter, then we could only identify that section from a higher common parent.